essential_reading/ReChunkingData.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
This notebook feels to me like it should go in `dataset_preprocessing/tutorials` - what do you think? It seems like a a walkthrough with instructional material explaining a way that we preprocess our data.

Reply:
agree. that thought hit me over dinner tonight... I just moved and pushed to this branch.